### ISSUE TITLE
the challange absoluteValuesSumMinization say sorted array

### ISSUE THREAD

**Issue by rollrodrig** [PizzaPokerGuy/100AlgorithmsChallenge#5]
the challange absoluteValuesSumMinization say sorted array but in the example you give an unsorted array  [2, 4, 7, 6, 6, 8].

**Comment by kat-ausmus** [comment]
I have the same question. The example is not a sorted array. 

the example 2,4,7,6,6 results to a `6` and not a `7`.  Because the input is not sorted properly.

**Comment by SukhdeepJSingh** [comment]
Yes, instructor made a mistake here, but you can assume a sorted input like [2, 4, 6, 6, 7, 8] or sort the given input yourself because median can only be calculated on sorted numbers.

**Comment by dobromyslov** [comment]
Agree. It's a weird mistake and it becomes even weirder when Dylan looks at the [2, 4, 7, 6, 6] returns 7 and says the algorythm is ok.

And another note is about median. When data series length is even then usually median is calculated as sum of two central elements divided by 2.
